continuous_integration/appveyor/setup_conda_environment.cmd

```diff
@@ -26,7 +26,8 @@ copy NUL %CONDA_PREFIX%\conda-meta\pinned
 @rem Install optional dependencies for tests
 %CONDA_INSTALL% numpy pandas cloudpickle distributed
 %CONDA_INSTALL% bcolz bokeh h5py ipython lz4 psutil pytables s3fs scipy
-%CONDA_INSTALL% -c conda-forge fastparquet snappy
+%CONDA_INSTALL% -c conda-forge fastparquet snappy zarr
+%CONDA_INSTALL% asciitree
```

Comment:
Why was this needed?

Reply:
As far as I know zarr is need for test_zarr and asciitree appears to be missing according to the appveyor error report following zarr installation.